models/items/interlibrary_loan/interlibrary_loan_item.rb

```diff
@@ -3,7 +3,7 @@ def initialize(parsed_response)
     super
     if @parsed_response["RequestType"] == "Article"
       @title = "#{@parsed_response["PhotoJournalTitle"] || ""} #{@parsed_response["PhotoArticleTitle"] || ""}"
-      @author = @parsed_response["PhotoArticleAuthor"] || ""
+      @author = @parsed_response["PhotoArticleAuthor"] || @parsed_response["PhotoItemAuthor"] || ""
```

Comment:
That is a good option. I don't know if there'd be a case where there'd be both an Article Author and a Photo Article Author. If it's one or the other this is good enough. If there could be two of them you could do an array join thing:

```
[@parsed_response['PhotoArticleAuthor'], @parsed_respons['PhotoItemAuthor'].reject { |e| e.to_s.empty? }.join('; ')
```

but I dunno. 

Also there should be a test for this.